ui/packages/catalog-ui-search/src/main/webapp/component/metacard/metacard.view.js

```diff
@@ -64,11 +66,37 @@ define([
         },
         handleRoute: function(){
             if (router.toJSON().name === 'openMetacard'){
-                this.$el.removeClass('is-hidden');
-            } else {
-                this.$el.addClass('is-hidden');
+                const metacardId = router.toJSON().args[0];
+                const queryForMetacard = new Query.Model({
+                    cql: cql.write({
+                        type: 'AND',
+                        filters: [{
+                            type: '=',
+                            value: metacardId,
+                            property: '"id"'
+                        }, {
+                            type: 'ILIKE',
+                            value: '*',
+                            property: '"metacard-tags"'
+                        }]
+                    }),
+                    federation: 'enterprise'
```

Comment:
❓ in what circumstances is this enterprise query sent out?

Reply:
This got moved, but it gets sent out when you visit the metacard route.